### ISSUE TITLE
source.changed() doesn't work in heatmap

### ISSUE THREAD

**Issue by ChenGuanglin0924** [openlayers/openlayers#10778]
```
const source = layer.getSource();
source.clear(true); // when param is 'false', it works
...
source.addFeatures(features);
source.changed(); // it doesn't work
```
It doesn't work only in ol.layer.heatmap, but works in ol.layer.vector.

**Comment by mike-000** [comment]
Heatmap uses the same renderer as a WebGLPoints layer so this looks like the same issue as #10721

**Comment by jahow** [comment]
Hi @ChenGuanglin0924, could you please confirm that your issue is fixed now that https://github.com/openlayers/openlayers/issues/10721 is merged? thanks!

**Comment by ChenGuanglin0924** [comment]
Where can i get the resources 'ol.js'? I can't find it in github.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Olivier Guyot"<notifications@github.com&gt;;
发送时间:&nbsp;2020年3月10日(星期二) 下午4:28
收件人:&nbsp;"openlayers/openlayers"<openlayers@noreply.github.com&gt;;
抄送:&nbsp;"GarascChan"<diyad7299@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [openlayers/openlayers] source.changed() doesn't work in heatmap (#10778)




Hi @ChenGuanglin0924, could you please confirm that your issue is fixed now that #10721 is merged? thanks!
